logger/log.go

```diff
@@ -362,22 +383,57 @@ func SetLogger(param ...string) {
 	conf := new(logConfig)
 	err := json.Unmarshal([]byte(c), conf)
 	if err != nil { //不是json，就认为是配置文件，如果都不是，打印日志，然后退出
-		// Open the configuration file
-		fd, err := os.Open(c)
-		if err != nil {
-			fmt.Fprintf(common.StdErr, "Could not open %s for configure: %s\n", c, err)
-			os.Exit(1)
-		}
+		//configuration file does not exist, use the default configuration
+		if f, err := os.Stat(c); err != nil || f.IsDir() {
+			var (
+				loglevel   = 5
+				loggerfile = filepath.Join(common.GetHomeDir(), ".sealer", "sealer.log")
```

Comment:
and how about we save log with more info? sealer-2011-1-1-13-13-13.log or something?